install_files/ansible-base/roles/validate/tasks/validate_gpg_info.yml

```diff
@@ -1,20 +1,24 @@
 ---
 
-- name: Validate GPG info.
+- name: Validate GPG fingerprints.
   assert:
     that:
```

Comment:
Great catch! Was surprised there's no `|abspath` filter in Ansible that would let you expand a path. Since Ansible looks in many different locations to expand a path, it'd be nice to introspect that a bit. Just occurred to me that a `lookup('file', filename)` would do the job nicely.